policy/modules/contrib/devicekit.te

```diff
@@ -278,6 +278,8 @@ term_use_all_inherited_terms(devicekit_power_t)
 
 auth_use_nsswitch(devicekit_power_t)
 
+logging_send_syslog_msg(devicekit_power_t)
```

Comment:
Strange as it may seem, logging is not in base.
Until recently no policy module had it in the optional block.

Reply:
OK, I changed it to an optional block.